### ISSUE TITLE
In-line comments are not supported

### ISSUE THREAD

**Issue by bsinou** [jaytaylor/go-hostsfile#5]
Hello,

First, thanks for the library, it is a nice helper.

Yet, I have found a limitation and would kindly ask if you might give a look: it seems like in-line comments are not handled as expected.
Typically, such lines in the `/etc/hosts` file (in GNU-Linux):

```
127.0.0.1	localhost #my-other-host my-other-commented-host
127.0.0.1	my.host # my-third-commented-host
```
returns such an array:

```
["localhost", "#my-other-host", "my-other-commented-host", "my.host", "#", "my-third-commented-host"]
```

where one would rather expect: 

```
["localhost", "my.host"]
```

**Comment by jaytaylor** [comment]
Thanks for reporting this, Bruno!

I agree, it seems like an area for improvement.  A PR with the fix + accompanying test case will be most welcome, and promptly merged!

**Comment by jaytaylor** [comment]
Resolved by PR #6
